backend/mcp/client.py

```
@@ -0,0 +1,498 @@
+        """
+        if not resource_type:
+            return {"error": "Resource type is required"}
+            
+        # Ensure params is a dictionary
+        if not isinstance(params, dict):
+            if isinstance(params, str):
+                try:
+                    # Try to parse as JSON
+                    import json
+                    params = json.loads(params)
+                except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)